.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,41 +1,41 @@
 ---
 name: 🐛 Bug Report
-about: Create a report to help us improve 🤔.
+about: Create an issue report.
 ---
 
-<!-- ⚠️ If you do not respect this template your issue will be closed. -->
+<!--
+Hello 👋 Thank you for submitting an issue.
 
-<!-- =============================================================================== -->
-<!-- ⚠️ If you are not using the current Strapi release, you will be asked to update. -->
-<!-- Please see the wiki for guides on upgrading to the latest release. -->
-<!-- =============================================================================== -->
+Before you start, please make sure your issue is understandable and reproducible.
+To me it readable please use a well markdown syntax.
```

Comment:
`To make your issue readable make sure you use valid Markdown syntax`

Reply:
I approved it but this seems unfixed still.